src/device/common/src/debug_frmwrk.c

```diff
@@ -275,7 +275,7 @@ void debug_frmwrk_init_clk(uint32_t Clock_Speed)
      */
     UART_ConfigStructInit(&UARTConfigStruct);
     // Re-configure baudrate to 115200bps
```

Comment:
Please update the comment to match the baud rate, thanks!

Reply:
Will do. thanks

Reply:
Done